src/middlewares/ContentTypeMiddleware.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {ContentTypeMiddleware} from "./ContentTypeMiddleware";
+import {IFetchRequest} from "./FetchMiddleware";
+
+describe("ContentTypeMiddleware", () => {
+    it("should add a content type application/json by default", () => {
+        const spy = jest.fn((options: IFetchRequest) => {
+            const contentType = options.headers && options.headers["content-type"];
+            expect(contentType).toBe("application/json");
```

Comment:
Is this `expect ` even gets called? Should we do [`expect.assertions`](https://jestjs.io/docs/en/asynchronous) to ensure that?

Reply:
I took that from another spec file, but let me verify that tonight :) (october just started, so...)

Reply:
![image](https://user-images.githubusercontent.com/3640284/94822736-30945900-042d-11eb-94a2-f99c1a600ac8.png)

it does fail if I remove the text/plain, so the expect does work, I took that as an example from another test,